src/kbemu.c

```diff
@@ -19,8 +19,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config.h"
 
 #ifdef USE_X11
+#include <math.h>  // round.
```

Comment:
spacenavd is a C89 codebase, C99 comments need to be changed before merging

Reply:
Sorry about that, I guess I should have read more about the project before writing my PR.

Reply:
Don't worry, I don't think it's documented anywhere.